packages/firebase_ml_vision/test/image_labeler_test.dart

```diff
@@ -33,7 +33,7 @@ void main() {
       test('processImage', () async {
         final List<dynamic> labelData = <dynamic>[
           <dynamic, dynamic>{
-            'confidence': 0.6,
+            'confidence': 1,
```

Comment:
Should we still have a test for `0.6` confidence?

Reply:
Same as above, 

Confidence here is now changed to the original value.